NitroxModel/DataStructures/NitroxInt3.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using NitroxModel.DataStructures.GameLogic;
+using ProtoBufNet;
+
+namespace NitroxModel.DataStructures
+{
+    /*
+     * NitroxInt3 allows NitroxModel to be decoupled from Assembly-csharp-firstpass
+     * as it is common code between subnautica and below zero
+     */
+    [Serializable]
+    [ProtoContract]
+    public class NitroxInt3
```

Comment:
A descriptive name like `Vec3I`may be preferable to smurf naming conventions?

Reply:
Normally we just stick "Nitrox" in front of the SN name which would fit in this case ("Int3").